spec/plans/util/retrieve_and_upload_spec.rb

```diff
@@ -5,7 +5,7 @@
   include BoltSpec::Plans
 
   it 'file needs downloaded and needs uploaded' do
-    expect_task('peadm::os_identification')
+    allow_task('facts').be_called_times(1).with_targets('local://localhost').always_return({ 'os' => {'family' => 'RedHat'} })
```

Comment:
This should pass now. Please keep in mind that it currently only tests for Linux. There were no tests for linux.